File: 3-strings/1238.c
#include<stdio.h>
#include<string.h>
//
int main(){
	int r;
	for(scanf("%i",&r);r>0;--r){
		char str1[55],str2[55];
		scanf("%s %s",str1,str2);
		for(size_t i=0;i<(strlen(str1)>strlen(str2)?strlen(str1):strlen(str2));++i){
			if(i<strlen(str1))
				printf("%c",str1[i]);
			if(i<strlen(str2))
				printf("%c",str2[i]);
		}
		printf("\n");
	}

	return 0;
}

File: 3-strings/1238.php
<?php
$REP=(int)readLine();
for($I=0;$I<$REP;++$I){
	list($STR1,$STR2)=explode(' ',readline());
	for($J=0;$J<max(strlen($STR1),strlen($STR2));++$J){
		if($J<strlen($STR1))
			echo $STR1[$J];
		if($J<strlen($STR2))
			echo $STR2[$J];
	}
	echo "\n";
}
?>
